nautobot/extras/templates/extras/plugins_list.html

```diff
@@ -3,10 +3,35 @@
 {% load static %}
 
 {% block content %}
+    <div class="row noprint">
+        <div class="col-md-12">
+            <ol class="breadcrumb">
+                {% block breadcrumbs %}
+                    <li><a href="{% url 'apps:apps_list' %}">Installed Apps</a></li>
+                {% endblock breadcrumbs %}
+            </ol>
+        </div>
+    </div>
+
     <div class="pull-right noprint">
+        <div class="btn-group">
+            <a class="btn btn-default set-plugins_list-display" data-display="tiles" href="" title="Tiles">
```

Comment:
nit: change these to `set-apps-display` perhaps?

Reply:
sure, not a problem 8be0cbd9d9f69b37dee06bb7c91b78eeecf09e9e